src/client/features/interactions/index.js

```diff
@@ -3,14 +3,32 @@ const h = require('react-hyperscript');
 const _ = require('lodash');
 const queryString = require('query-string');
 const Loader = require('react-loader');
-
+const hideTooltips = require('../../common/cy/events/click').hideTooltips;
+const removeStyle= require('../../common/cy/manage-style').removeStyle;
 const make_cytoscape = require('../../common/cy/');
 const interactionsStylesheet= require('../../common/cy/interactions-stylesheet');
 const { ServerAPI } = require('../../services/');
-
+const FilterMenu= require('./filter-menu');
 const { BaseNetworkView } = require('../../common/components');
 const { getLayoutConfig } = require('../../common/cy/layout');
 
+const filterMenuId='filter-menu';
+const interactionsConfig={
+  toolbarButtons: BaseNetworkView.config.toolbarButtons.concat({
+    id: 'filter',
+    icon: 'filter_list',
+    type: 'activateMenu',
+    menuId: filterMenuId,
+    description: 'Filter interaction types'
+  }),
+  menus: BaseNetworkView.config.menus.concat({
+    id: filterMenuId,
+    width: 25, //%
```

Comment:
Does it make sense to have it as a number?  Would it be better to just let it be '25%'?

Reply:
The value of the width also gets removed from the width of the graph element to make sure it aligns to the edge of the menu still.